starters/blog/package.json

```diff
@@ -50,6 +53,7 @@
     "develop": "gatsby develop",
     "start": "npm run develop",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\""
+    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
+    "format": "prettier --write src/**/*.{js,jsx}"
```

Comment:
Nit, and I'm sorry already--but can we move this up after develop? (for all starters)

It's nice to keep these consistent and well-structured.

Once done we can get this merged!

Reply:
Sure. No problem.

Reply:
Fixed it.